pt/lessons/basics/modules.md

```diff
@@ -9,7 +9,7 @@ Sabemos por experiência o quanto é incontrolável ter todas as nossas funçõe
 
 ## Modulos
```

Comment:
I was just reviewing and noticed some differences throughout the text. I believe that if we want to keep the word "módulo" for the Portuguese version, maybe then we can just update this title to "Módulo".

If you check the translation for "Structs", we don't translate it to "estruturas" anymore. In this case I find it more appropriate to not translate since the word *struct* is also used in the language.

Thoughts @pragmaticivan ?

Reply:
I prefer using `Módulo` as well.

Reply:
Even change `Composição` to `Módulos`